libraries/AP_Notify/RGBLed.cpp

```diff
@@ -262,7 +262,7 @@ void RGBLed::update_colours(void)
             // fast flashing green if disarmed with GPS 3D lock and DGPS
             // slow flashing green if disarmed with GPS 3d lock (and no DGPS)
             // flashing blue if disarmed with no gps lock or gps pre_arm checks have failed
-            bool fast_green = AP_Notify::flags.gps_status >= AP_GPS::GPS_OK_FIX_3D_DGPS && AP_Notify::flags.pre_arm_gps_check;
+            bool fast_green = AP_Notify::flags.gps_status > AP_GPS::GPS_OK_FIX_3D_DGPS && AP_Notify::flags.pre_arm_gps_check;
```

Comment:
Allow me to explain my reasoning here, and we can see if I'm mistaken! When I originally added the fast-flash capability, the intent was to demonstrate when the GPS was using additional differential observations *that we supply*, so it was an easy way to check that the end-to-end system was working. In the meantime, it looks as if satellite-based augmentation such as ublox' SBAS receiver now also reports a DGPS status, resulting in a fast-flash.  If we think this is useful, then let's keep it as is. I would argue for moving back to the original indication: when you're getting the fast flash, that means your GPS is using more than just a satellite-received signal, but is getting augmentation from another source.